app/src/pages/inside/common/launchFiltersToolbar/filterList/filterItem/filterTooltipIcon/filterTooltipIcon.scss

```diff
@@ -0,0 +1,7 @@
+.filter-tooltip-icon {
+  position: relative;
+  top: 2px;
+  display: inline-block;
+  width: 18px;
```

Comment:
```
width: 16px;
height: 16px;
```
as in `app/src/pages/inside/common/launchFiltersToolbar/filterList/filterItem/filterItem.scss#25`

Please also move icon svg style from the file above to this file